src/Analysis/Ast/Impl/Specializations/Typing/Types/GenericTypeParameter.cs

```diff
@@ -85,15 +107,16 @@ public GenericTypeParameter(string name, IPythonModule declaringModule, IReadOnl
             var args = argSet.Arguments;
             var constraintArgs = argSet.ListArgument?.Values ?? Array.Empty<IMember>();
 
-            var name = (args[0].Value as IPythonConstant)?.GetString();
+            var name = argSet.GetArgumentValue<IPythonConstant>("name")?.GetString();
             var constraints = constraintArgs.Select(a => {
                 // Type constraints may be specified as type name strings.
                 var typeString = (a as IPythonConstant)?.GetString();
```

Comment:
Looks like `(a as IPythonConstant)?.GetString()` starts getting popular, may be worth making an extension.

Reply:
Sure